vivisect/base.py

```diff
@@ -644,6 +647,14 @@ def _mcb_Platform(self, name, value):
         if defcall:
             self.setMeta('DefaultCall', defcall)
 
+    def _mcb_FileBytes(self, name, value):
+        if not self.parsedbin:
+            byts = viv_parsers.uncompressBytes(value)
+            fmt = viv_parsers.guessFormat(byts)
+            parser = viv_parsers.getBytesParser(fmt)
+            if parser:
+                self.parsedbin = parser(byts)
```

Comment:
so if i'm reading this correctly, the workspace will both have a filemeta entry "FileBytes" with the original file in it (+1), *and* on load, it will load the parser as it was on initial analysis (basically) and keep it in the operational namespace? (+1)

the only possible concern is loading the workspace over the network (Server or SharedWorkspace) will take twice as long.
we may want to have the server *optionally* not send the VWE_ event for `setFileMeta(<file>, 'FileBytes, <bytez>)`, but i don't know if that's necessary for this PR to be merged...

this approach will mean that any changes to the parser will make this `vw.parsedbin` differ from what was originally loaded... but i'm not really worried about that either.

Reply:
To the first point, almost. The FileBytes meta will be the original bytes of the file saved off. However, on loading those bytes, if the self.parsedbin property hasn't been already filled in by the parser, it'll load that up. So really parsedbin should reflect whatever the current state of the Elf/PE submodules are (parser in this case doesn't refer to `vivisect.parsers.pe/elf` but instead to `PE/Elf` modules we've got. So it is possible that `vw.parsedbin` can differ from what it originally was. But I'm not particularly worried about that either since those modules don't move that fast.

So I'm against getting into special casing what we send over the remote and what we don't. I feel like that's a rabbit hole that I don't want to introduce, especially in this PR. It's also a one time cost